server/src/test/java/org/apache/iotdb/db/utils/MemUtilsTest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.db.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.iotdb.db.exception.metadata.IllegalPathException;
+import org.apache.iotdb.db.metadata.PartialPath;
+import org.apache.iotdb.db.qp.physical.crud.InsertTabletPlan;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.write.record.TSRecord;
+import org.apache.iotdb.tsfile.write.record.datapoint.*;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class MemUtilsTest {
+
+  @Test
+  public void recordSizeTest() throws IllegalPathException {
+    Assert.assertEquals(12, MemUtils.getRecordSize(TSDataType.INT32, 10, true));
+    Assert.assertEquals(16, MemUtils.getRecordSize(TSDataType.INT64, 10, true));
+    Assert.assertEquals(12, MemUtils.getRecordSize(TSDataType.FLOAT, 10.0, true));
+    Assert.assertEquals(16, MemUtils.getRecordSize(TSDataType.DOUBLE, 10.0, true));
+    Assert.assertEquals(8, MemUtils.getRecordSize(TSDataType.TEXT, "10", false));
+
+    PartialPath device = new PartialPath("root.sg.d1");
+    String[] measurements = {"s1", "s2", "s3", "s4", "s5"};
+    List<Integer> dataTypes = new ArrayList<>();
+    int sizeSum = 0;
+    dataTypes.add(TSDataType.INT32.ordinal());
+    sizeSum += 8 + TSDataType.INT32.getDataTypeSize();
+    dataTypes.add(TSDataType.INT64.ordinal());
+    sizeSum += 8 + TSDataType.INT64.getDataTypeSize();
+    dataTypes.add(TSDataType.FLOAT.ordinal());
+    sizeSum += 8 + TSDataType.FLOAT.getDataTypeSize();
+    dataTypes.add(TSDataType.DOUBLE.ordinal());
+    sizeSum += 8 + TSDataType.DOUBLE.getDataTypeSize();
+    dataTypes.add(TSDataType.TEXT.ordinal());
+    sizeSum += TSDataType.TEXT.getDataTypeSize();
+    InsertTabletPlan insertPlan = new InsertTabletPlan(device, measurements, dataTypes);
+    Assert.assertEquals(sizeSum, MemUtils.getRecordSize(insertPlan, 0, 1, false));
+  }
+
+  @Test
+  public void TsRecordSizeTest() {
```

Comment:
`TsRecordSizeTest` -> `TsRecordMemTest`. And add javadoc for this test, it tests getDataPointMem() and getTsRecordMem().